slurm.spec

```diff
@@ -612,8 +612,6 @@ rm -rf $RPM_BUILD_ROOT
 %defattr(-,root,root)
 %dir %attr(0755,root,root) %{_prefix}/include/slurm
 %{_prefix}/include/slurm/*
-%{_libdir}/libpmi.la
-%{_libdir}/libpmi2.la
```

Comment:
Should these files also be deleted in "Delete unpackaged files" section? Otherwise you may get an error about unpackaged files during rpmbuild.

Also I note the NEWS entry says "remove pmi and pmi2 libraries from devel package", that might be better worried as "remove pmi and pmi2 *libtool archive* files from devel package"